### PR TITLE
Vladimir/3670

### PR DESCRIPTION
**Pull request by EfimovVladimir**

### CODE REVIEW

File: test/protractorConf.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ var config = {
     'tests/bugs.js',
     'tests/vim-mode.js',
     'tests/charting-tutorial.js',
-    'tests/category-plot-tutorial.js'
+    'tests/category-plot-tutorial.js',
+
```
Comment (scottdraves) [inline test/protractorConf.js]: no blank line here

File: test/tests/heatmap-tutorial.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    it('Custom size, no tooltips', function () {
+        beakerPO.checkPlotIsPresent(5);
+        beakerPO.checkSize(beakerPO.getPlotSvg(5, 0), 900, 300);
+
+        var nonexistent = beakerPO.getPlotSvg(5, 0).all(By.css('.plot-resp'));
+        expect(nonexistent.getText()).toEqual([ ]);
+    });
+
+});
```
Comment (scottdraves) [inline test/tests/heatmap-tutorial.js:79]: files should end in newline

File: .gitignore [hunk 3]
```diff
@@ -127,14 +127,3 @@ core/src/main/web/vendor/bower_components/request-frame/demo
 core/src/main/web/vendor/bower_components/request-frame/src
 core/src/main/web/vendor/bower_components/request-frame/gulpfile.js
 core/src/main/web/vendor/bower_components/request-frame/.gitignore
-
-
-core/src/main/web/vendor/bower_components/jquery-throttle-debounce/docs
```
Comment (scottdraves) [inline .gitignore:132]: why are you removing these?

Comment (EfimovVladimir) [inline .gitignore:132]: Accidentally. I've reverted this in next commit

**Comment by tomca32** [comment]
@EfimovVladimir - can you associate your email address that you use for git with your github account? Without it github doesn't associate these commits with your account.

You should be able to do it here: https://github.com/settings/emails

**Comment by tomca32** [comment]
kick jenkins

**Comment by scottdraves** [comment]
kick jenkins

**Pull request merged**